ThirdPartyAdapters/vungle/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Vungle Android Mediation Adapter Changelog
 
+#### Version 6.7.1.0
+- Verified compatibility with Vungle SDK 6.7.1.
+- Various Bug Fixes
+- Removed Vungle#setUserLegacyID API
```

Comment:
We need to remove mention of setUserLegacyID, we dont want to highlight this even for deprecation